src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/toolbar.css.twig

```diff
@@ -218,8 +216,7 @@
 }
 
 .sf-toolbar-block .sf-toolbar-icon .sf-toolbar-status {
-    border-radius: 12px;
-    border-bottom-left-radius: 0;
+    border-radius: 12px 12px 12px 0;
```

Comment:
:-1: on this one as well, previous one is easier to understand and easier to maintain (e.g. when changing the radius).